web_ui/src/shared/components/media-items-list/media-items-list.component.tsx

```diff
@@ -64,6 +64,19 @@ export const MediaItemsList = <T extends object>({
               preserveAspectRatio: true,
           };
 
+    const ref = useRef<HTMLDivElement | null>(null);
+    useLoadMore({ onLoadMore: endReached }, ref);
+
+    const container = ref?.current?.firstElementChild;
+
+    useScrollToTargetItem({
+        gap: config.gap,
+        container,
+        targetIndex: scrollToIndex,
+        dependencies: [scrollToIndex, viewMode, container],
+        callback: (top) => ref.current?.scrollTo({ top, behavior: 'smooth' }),
```

Comment:
I'd expect react aria to have a specific function to focus on a virtualized element, which then triggers the scrolling. Did you check if there's not a more explicit way to do this?

Reply:
I couldn't find a specific helper or handler in either the react-aria utilities or the component itself